components/SearchResult.tsx

```diff
@@ -1,9 +1,11 @@
 import { useQuery } from "@tanstack/react-query";
 import Result from "./Result";
 import { Album, SearchResponseSchema } from "@/schemas/collections.schemas";
+import Icon from "./Icon";
+import ResultLoadingUi from "./ResultLoadingUi";
 
 export default function SearchResult({ searchValue }: { searchValue: string }) {
-  const { data, isError, isLoading } = useQuery({
+  const { data, isError, isLoading, isSuccess } = useQuery({
```

Comment:
```suggestion
  const { data, isError, isLoading } = useQuery({
```

since you aren't using this